<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>carsFormWidget</class>
 <widget class="QWidget" name="carsFormWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nuevo Carro</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Número Serial</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Marca</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brandTextLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="serialTextLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveCarsButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelCarsButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="modelTextLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Modelo</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="transmisionTextLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Transmisión</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="priceTextLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yearTextLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Año</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
